src/proxyless/cookie-provider.ts

```diff
@@ -31,12 +33,17 @@ export class CdpCookieProvider extends CookieProviderBase implements CookieProvi
         return (matchCollection(cookies, externalCookies) as Cookie[]).map(this._cdpCookieToExternalCookie);
     }
 
-    async setCookies (cookies: CookieOptions[], url: string): Promise<void> {
-        const client = await this._getCdpClient();
+    async setCookies (cookies: string | string[] | CookieOptions[], url: string): Promise<void> {
```

Comment:
please add some description in the PR initial post.
Why we need to pass `string` or `string[]`